src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/DefaultMotion.tsx

```diff
@@ -19,32 +19,33 @@ interface DefaultMotionProps {
   actionData: ColonyAction;
 }
 
-const DefaultMotion = ({ actionData }: DefaultMotionProps) => {
-  const { colony } = useColonyContext();
+const DefaultMotion = ({
+  actionData,
+  actionData: { motionData },
+}: DefaultMotionProps) => {
+  const { colony, showBanner, setShowBanner, showMotionHeading } =
+    useDefaultMotion(motionData);
```

Comment:
I feel iffy importing the colony from this hook.

I know it imports it properly under the hood, but from a eagle eye view it may looks like both `useDefaultMotion` and `useColonyContext` export the `colony` data object, without knowing for sure if there are differences between them.

I would stiill make use of the `colony` data internally in `useDefaultMotion`, but when used in components, I would import the colony data from the standard `useColonyContext`